File: src/main/java/com/melonbase/microquark/repo/data/DataRoot.java
package com.melonbase.microquark.repo.data;

import java.util.HashSet;
import java.util.Set;

public class DataRoot {

  private Set<Volksabstimmung> volksabstimmungen = new HashSet<>();

  public Set<Volksabstimmung> getVolksabstimmungen() {
    return volksabstimmungen;
  }

  public void setVolksabstimmungen(Set<Volksabstimmung> volksabstimmungen) {
    this.volksabstimmungen = volksabstimmungen;
  }
}

File: src/main/java/com/melonbase/microquark/repo/data/DataRoot.kt
package com.melonbase.microquark.repo.data;

import java.util.HashSet;
import java.util.Set;

public class DataRoot {

  private Set<Volksabstimmung> volksabstimmungen = new HashSet<>();

  public Set<Volksabstimmung> getVolksabstimmungen() {
    return volksabstimmungen;
  }
}
